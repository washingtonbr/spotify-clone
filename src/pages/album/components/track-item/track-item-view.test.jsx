import React from 'react';
import renderer from 'react-test-renderer';
import TrackItemView from './track-item-view';

jest.mock('../../../../utils/time', () => ({
  millisToMinutesAndSeconds: jest.fn(),
}));

describe('TrackItemView', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(
        <TrackItemView
          index={1}
          name="Premonition - Intro"
          duration="173893"
        />,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
