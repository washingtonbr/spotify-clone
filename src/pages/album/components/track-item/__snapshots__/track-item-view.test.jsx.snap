// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TrackItemView renders correctly 1`] = `
<li
  className="sc-AxjAm ernjFJ"
>
  <div
    className="sc-AxirZ iXctlD"
  >
    2
  </div>
  <div
    className="sc-AxiKw jeFTTI"
  >
    Premonition - Intro
  </div>
  <div
    className="sc-AxhCb hNGpi"
  />
</li>
`;
